src/entities.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
+from __future__ import print_function
+import json
+import re
+
+### Declare headers
+header_a = """/*
+ * Generated file - do not edit directly.
+ * This file was generated from:
+ *     https://html.spec.whatwg.org/entities.json
+ * by means of entities.py in the libgd src dir
+*/
+"""
+
+header_b = """
+#include "entities.h"
+
+const struct entities_s entities[NR_OF_ENTITIES] = {"""
+
+header_c = """/*
+ * Generated file - do not edit directly.
+ * This file was generated from:
+ *     https://html.spec.whatwg.org/entities.json
+ * by means of entities.py in the libgd src dir
+*/
+#ifndef GD_ENTITIES_H
+
+#include <stdlib.h>
+#include <stdint.h>
+
+//html entity strings are entity prefix + string + suffix limited
+//hex and dec should be limited to current unicode spec + entity prefix + suffix
+"""
+
+header_d = """
+struct entities_s{
+                  	char *name;
```

Comment:
only indent with a single tab here (and below)

Reply:
looks like these members are still using spaces for indentation.  we just want one tab.